Tasks/ArchiveFiles/archivefiles.ts

```diff
@@ -1,7 +1,7 @@
 import path = require('path');
 import tl = require('vsts-task-lib/task');
 
-var repoRoot: string = tl.getVariable('build.sourcesDirectory');
+var repoRoot: string = tl.getVariable('System.DefaultWorkingDirectory');
```

Comment:
In the DecriptFile task below we can see even more options:
```
var cwd = tl.getInput('cwd')  
       || tl.getVariable('build.sourceDirectory')  
       || tl.getVariable('build.sourcesDirectory')  
       || tl.getVariable('System.DefaultWorkingDirectory') 
```
IMO, either we should use those in all tasks or remove ```tl.getVariable('build.sourceDirectory') ``` there.